panels/zwave/ha-panel-zwave.html

```diff
@@ -112,6 +225,209 @@
+    var configData = [];
+    this.hass.callApi('GET', 'zwave/config/' + this.nodes[selectedNode].attributes.node_id).then(function (configs) {
+      Object.entries(configs).forEach(([key, value]) => {
+        configData.push({ key, value });
+      });
+      this.config = configData;
+//      console.log(this.config);
+    }.bind(this));
+    var groupData = [];
+    this.hass.callApi('GET', 'zwave/groups/' + this.nodes[selectedNode].attributes.node_id).then(function (groups) {
+      Object.entries(groups).forEach(([key, value]) => {
+        groupData.push({ key, value });
+      });
+      this.groups = groupData;
+//      console.log(this.groups);
+    }.bind(this));
+    var userCodes = [];
+    this.hasNodeUserCodes = false;
+    this.notifyPath('hasNodeUserCodes');
+    this.hass.callApi('GET', 'zwave/usercodes/' + this.nodes[selectedNode].attributes.node_id).then(function (usercodes) {
+      Object.entries(usercodes).forEach(([key, value]) => {
+        userCodes.push({ key, value });
+      });
+      this.userCodes = userCodes;
+      if (Object.keys(userCodes).length === 0) {
+        this.hasNodeUserCodes = false;
+      } else {
+        this.hasNodeUserCodes = true;
+      }
+//      console.log(this.hasNodeUserCodes);
+      this.notifyPath('hasNodeUserCodes');
+//      console.log('fetched usercodes', this.userCodes);
+    }.bind(this));
+  },
+
+  computeSelectedEntityAttrs: function (selectedEntity) {
+    if (selectedEntity === -1) return 'No entity selected';
+    var entityAttrs = this.entities[selectedEntity].attributes;
+    var att = [];
+    Object.entries(entityAttrs).forEach(([key, value]) => {
+      att.push(key + ': ' + value);
+    });
+//    console.log('Entity attributes: ', att);
+    return att.sort();
+  },
+
+  computeSelectCaption: function (stateObj) {
+    return window.hassUtil.computeStateName(stateObj) + ' (Node:' +
+      stateObj.attributes.node_id + ' ' +
+      stateObj.attributes.query_stage + ')';
+  },
+
+  computeSelectCaptionEnt: function (stateObj) {
+    return window.hassUtil.computeStateName(stateObj);
+  },
+
+  computeIsNodeSelected: function () {
+    return (!this.nodes || this.selectedNode === -1);
+  },
+
+  computeIsEntitySelected: function (selectedEntity) {
+//    console.log('Entity selected: ', (selectedEntity === -1), selectedEntity);
+    return (selectedEntity === -1);
+  },
+
+  computeNodeServiceData: function (selectedNode) {
+    return { node_id: this.nodes[selectedNode].attributes.node_id };
+  },
+
+  computeGetNodeName: function (selectedNode) {
+    if (this.selectedNode === -1 ||
+      !this.nodes[selectedNode].entity_id) return -1;
+    var str = (this.nodes[selectedNode].entity_id);
+    var name = str.replace('zwave.', '');
+//    console.log('Node name: ', name);
+    return name;
+  },
+
+  computeNodeNameServiceData: function (newNodeNameInput) {
+//    console.log('Rename node:', JSON.stringify({ node_id: this.nodes[this.selectedNode].attributes.node_id,
```

Comment:
For the future, you don't want to stringify before logging. That way you can actually play with the object.